content/blog/2023/06 /02

```diff
@@ -0,0 +1,17 @@
+title: "The journey of Letica and development assessments - Research Call with Jonathan Reams"
+created: 2023-06-02
+categories: 
+  - "research-call"
+authors: 
+  - Rufuspollock
+
+>This call forms part of our monthly Research Call events. To get involved with this and our community calls please join the Life Itself [Whatsapp Chat](https://chat.whatsapp.com/JNJCTZugNQn)
+
+**When:** 15:00CET, Friday June 9th, 2023
```

Comment:
spacing after 15:00 i.e. `15:00 CET` also i would recommend saying `15:00 European` - strictly we aren't on CET atm so this time is actually wrong.